app/_plugins/generators/sitemap.rb

```diff
@@ -0,0 +1,67 @@
+module Sitemap
+  class Generator < Jekyll::Generator # rubocop:disable Metrics/ClassLength
+    priority :lowest
+
+    def generate(site)
+      # What's our latest version?
+      latest = site.data['versions'].reject { |v| v['release'] == 'dev' }.last
```

Comment:
versions.yml has lost the `latest` boolean that used to exist. This `latest` boolean is super useful as it enables us to ship the docs for the release during stabilization week without making it indexed by Google and still show the real current one.

In other words as soon as we'd cut the 2.0 release the 2.0 docs would start to exist but `latest` on the website would still be `1.8`.
Does that make sense?

Reply:
This was discussed live and we decided to do this in the next PR :)

Reply:
It makes sense - will re-add `latest` in a followup PR